tools/Makefile

```diff
@@ -136,8 +134,9 @@ $(foreach tool, $(tools-y), $(if $(wildcard $(curdir)/$(tool)/patches),$(eval $(
 
 $(foreach tool, $(filter-out xz,$(tools-y)), $(eval $(curdir)/$(tool)/compile += $(curdir)/xz/compile))
 
-# make any tool depend on tar, xz and patch to ensure that archives can be unpacked and patched properly
-tools-core := tar xz patch
+# make any tool depend on libdeflate, patch, tar, and xz to ensure that archives
+# can be unpacked and patched properly
+tools-core := libdeflate patch tar xz
```

Comment:
Doesn't belong in this commit. You're not changing the unpack routine.

Reply:
Are you telling me order matters? You put libdeflate as part of the tools-core, so I took that over, and fixed sorting as part of this.

Reply:
should we change this list like the tools one (one tool per line) for consistency and better tracking?

Reply:
no. this PR doesn't force unpacking with libdeflate. why would it be moved to tools-core?

Reply:
That's something I have no idea for; but I took this from your set of changes @neheb